### DISCUSSION TITLE
OG Image Generation App Crashes

### DISCUSSION THREAD

**Discussion by semrebyrk** [vercel/community#1345] [Help]
Hello,

I d like to implement OG Image Generation in my Next JS application. But after I create "pages/api/og.tsx" the app crashes and doesn't work. The only difference I made is I wrote my code in jsx format, so I had to change the file format.

Thanks for any response in advance

**Comment by amyegan** [comment]
Hi @semrebyrk! Do you see any addition error information in your [Function Logs](https://vercel.com/docs/concepts/deployments/logs#function-logs)?
